metis/lib/models/bucket.rb

```diff
@@ -15,6 +15,8 @@ class Bucket < Sequel::Model
       guest: 3
     }
 
+    READ_ONLY_BUCKETS = ['magma']
```

Comment:
I will refactor this a bit next week to put the "is this a read-only bucket" into the `Metis::Bucket` class, so it's cleaner and should then automatically extend to any other type of "app" bucket, too. i.e. `vulcan` owned `vulcan` bucket, if we want to manage the cache in Metis.